GVRf/Framework/framework/src/main/jni/objects/components/render_data.cpp

```diff
@@ -219,7 +219,7 @@ std::string RenderData::getHashCode()
         render_data_string.append(to_string(stencilOpSfail_));
         render_data_string.append(to_string(stencilOpDpfail_));
         render_data_string.append(to_string(stencilOpDppass_));
-
+        render_data_string.append(to_string(mesh_->getVertexBuffer()->getDescriptor()));
```

Comment:
Can mesh_ or the result of getVertexBuffer() be nullptr?

Reply:
No since Vertex buffer is created in constructor of mesh.